### ISSUE TITLE
Add get mail info ROS service in mysql wrapper

### ISSUE THREAD

**Issue by etsardou** [rapp-project/rapp-platform#218]
Should take the username and return the info

**Comment by thanoski** [comment]
addressed in commit 6de2244cc242fe8926d5cfc55237b7dfe070dd5c

**Comment by athallas** [comment]
@thanoski 

Are lines 388-389 in 6de2244 supposed to be like this? :
```python
res.password=str(result_set[0][1])
res.password=str(result_set[0][2])
```
According to the query, the correct assignment seems to be:
```python
res.password=str(result_set[0][1])
res.server=str(result_set[0][2])
```

Also close this issue after it has been merged into the `devel` branch.

**Comment by etsardou** [comment]
Deprecated due to #220

**Comment by thanoski** [comment]
yes it was a typo, addressed in 6690ad775064779306730616c95c7ab35026389f
